etc/openqa/openqa.ini

```diff
@@ -144,6 +144,9 @@ blocklist = job_grab job_done
 # Specify the number of screenshot ID ranges (with a size as configured by screenshot_cleanup_batch_size) to process in a single Minion
 # job (reduce to avoid Minion jobs from running very long and possibly being interrupted, increase to reduce the number of Minion jobs)
 #screenshot_cleanup_batches_per_minion_job = 450
+# Extends the job result cleanup to ensure the partition results are stored on does not become too full
+# (still experimental, relies on df)
+#results_min_free_disk_space_percentage = 0
```

Comment:
I wonder if it wouldn't be more consistent to go with gigabytes like we do for asset sizes. `_percentage` is clear enough in that sense. Just wanted to mention that it seems like a deviation.

Reply:
When it comes to preserving a minimum amount of free disk space tools usually let you specify a percentage so I preferred this.